bin/pre-install.js

```diff
@@ -0,0 +1,99 @@
+// The "userAgent" is something like 'yarn/0.28.1 npm/? node/v8.4.0 linux x64'
```

Comment:
We need the license pre-amble. Could you also write some an intro comment explaining why this file exists.

Reply:
done